manifests/initrd.pp

```diff
@@ -0,0 +1,21 @@
+#
+# == Class: kmod::initrd
+#
+# Reconfigures the initramfs image after modifying kernel module configuration.
+#
+#
+class kmod::initrd(Optional[String] $cmd = $facts.kmod_update_initrd_cmd){
+  if defined($cmd) {
+    exec { "${module_name}: Update initramfs image.":
+      command     => $cmd,
+      refreshonly => true,
+      subscribe   => [
+        Class['kmod::alias'],
```

Comment:
These classes don't exist, so I don't think this will work.

Reply:
Ah, good catch. This would need to be converted to a resource collector.

Reply:
Yeah, I wasn't sure how to code that properly.